assets/airports/eidw.json

```diff
@@ -883,124 +1076,361 @@
+                "28": [["SUTEX", "A210|S350"], ["DIRUM", "A170|S300"], ["KEPOR", "A130|S270"], ["ARVOK", "A100|S250"], ["SORIN", "A70|S230"], "SIVNA", "SUGAD", "DW704", "DW705", ["DW706", "A70"], ["SOPEP", "A30"], ["LAPMO", "A30|S180"]],
+                "16": [["SUTEX", "A250|S350"], ["DIRUM", "A210|S300"], ["KEPOR", "A170|S270"], ["OTNER", "A150|S250"], ["DWN27", "A80"], ["ULTAG", "A60|S220"]],
+                "34": [["SUTEX", "A200|S350"], ["DIRUM", "A160|S280"], ["KEPOR", "A120|S250"], ["ARVOK", "A90|S240"], ["SORIN", "A60"]],
+                "10X": [["SUTEX", "A160|S270"], ["DIRUM", "A120|S250"], ["KANUS", "A90|S230"], ["BABON", "A70"], "BIVDI", "DW754", ["DW755", "A70"], ["OSLEX", "A50"]],
+                "28X": [["SUTEX", "A210|S350"], ["DIRUM", "A170|S300"], ["KEPOR", "A130|S270"], ["ARVOK", "A100|S250"], ["SORIN", "A70|S230"], "SIVNA", "SUGAD", "DW704", "DW705", ["DW706", "A70"], ["SOPEP", "A30"], ["LAPMO", "A30|S180"]],
+                "16X": [["SUTEX", "A250|S350"], ["DIRUM", "A210|S300"], ["KEPOR", "A170|S270"], ["OTNER", "A150|S250"], ["DWN27", "A80"], ["ULTAG", "A60|S220"]],
+                "34X": [["SUTEX", "A200|S350"], ["DIRUM", "A160|S280"], ["KEPOR", "A120|S250"], ["ARVOK", "A90|S240"], ["SORIN", "A60"]]
+            }
+        },
+        "OSGAR": {
+            "icao": "OSGAR",
+            "name": "Osgar",
+            "entryPoints": {
+                "SHA": ["SHA", ["TOMTO", "A350-"], ["TORLU", "A280"]]
+            },
+            "rwy": {
+                "10": [["OSGAR", "A190|S300"], ["DIRUM", "A120|S250"], ["KANUS", "A90|S230"], ["BABON", "A70"], "BIVDI", "DW754", ["DW755", "A70"], ["OSLEX", "A50"]],
+                "28": [["OSGAR", "A240|S350"], ["DIRUM", "A170|S300"], ["KEPOR", "A130|S270"], ["ARVOK", "A100|S250"], ["SORIN", "A70|S230"], "SIVNA", "SUGAD", "DW704", "DW705", ["DW706", "A70"], ["SOPEP", "A30"], ["LAPMO", "A30|S180"]],
+                "16": [["OSGAR", "A280|S350"], ["DIRUM", "A210|S300"], ["KEPOR", "A170|S270"], ["OTNER", "A150|S250"], ["DWN27", "A80"], ["ULTAG", "A60|S220"]],
+                "34": [["OSGAR", "A230|S350"], ["DIRUM", "A160|S280"], ["KEPOR", "A120|S250"], ["ARVOK", "A90|S240"], ["SORIN", "A60"]],
+                "10X": [["OSGAR", "A190|S300"], ["DIRUM", "A120|S250"], ["KANUS", "A90|S230"], ["BABON", "A70"], "BIVDI", "DW754", ["DW755", "A70"], ["OSLEX", "A50"]],
+                "28X": [["OSGAR", "A240|S350"], ["DIRUM", "A170|S300"], ["KEPOR", "A130|S270"], ["ARVOK", "A100|S250"], ["SORIN", "A70|S230"], "SIVNA", "SUGAD", "DW704", "DW705", ["DW706", "A70"], ["SOPEP", "A30"], ["LAPMO", "A30|S180"]],
+                "16X": [["OSGAR", "A280|S350"], ["DIRUM", "A210|S300"], ["KEPOR", "A170|S270"], ["OTNER", "A150|S250"], ["DWN27", "A80"], ["ULTAG", "A60|S220"]],
+                "34X": [["OSGAR", "A230|S350"], ["DIRUM", "A160|S280"], ["KEPOR", "A120|S250"], ["ARVOK", "A90|S240"], ["SORIN", "A60"]]
+            }
         }
     },
     "spawnPatterns": [
```

Comment:
From [Wikipedia](https://en.wikipedia.org/wiki/Dublin_Airport), the 2016 stats indicate 207,520 aircraft movements. IAW the [traffic level specs](https://github.com/openscope/openscope/blob/develop/documentation/airport-file-standards.md), this should fall between 23acph and 72acph. Currently, there's a total of 43acph.

This is good to go in that regard. :+1: